test/e2e/tests/test_replicationgroup.py

```diff
@@ -140,18 +140,52 @@ def rg_cmd_fromsnapshot(bootstrap_resources, make_rg_name, make_replication_grou
 
 
 @pytest.fixture(scope="module")
-def rg_cmd_update_input(make_rg_name):
+def rg_cme_uneven_shards_input(make_rg_name):
     return {
-        "RG_ID": make_rg_name("rg-cmd-update"),
+        "RG_ID": make_rg_name("rg-cme-uneven-shards"),
+        "NGID1": '"1111"',
+        "NGID2": '"2222"'
```

Comment:
instead of all the double-quote trimming, why not just name these `ng-1` and `ng-2`?

Reply:
From what I tried last time, I think there's some weird constraint on this field that it has to be a string of numeric characters (although I'd have to double check that).

When these inputs are injected into the appropriate `.yaml`, the outer quotes are stripped and the inner double quotes are left to clarify that the value is a string instead of an integer.